content/en/docs/armory-admin/armory-agent/_index.md

```diff
@@ -0,0 +1,89 @@
+---
+title: "Armory Agent for Kubernetes"
+linkTitle: "Armory Agent"
+weight: 4
+description: >
+  How the Armory Agent works
+---
+
+The Armory Agent is a lightweight service that connects back to Spinnaker's Clouddriver service and streams changes from the Kubernetes clusters it is configured to watch.
+
+> The Agent connects back to a new port opened on Clouddriver using gRPC.
+
+
+## Topologies
+
```

Comment:
We present three options but don't discuss what decision criteria to use to determine which one works best for the user. Line 22 kind of starts talking about this, but I still don't understand why I would use the Agent at all.

Reply:
I was hoping to get started with this since the modes are referenced in other places and improve later. I wrote a blog post to that end. I'd suggest to revisit later instead of providing a botched explanation.